### DISCUSSION TITLE
Error in cheerio.fromURL

### DISCUSSION THREAD

**Discussion by gerardocrr** [cheeriojs/cheerio#4022] [Q&A]
I have an API made with Hono, I was testing the fromURL method with the direct code from the [documentation ](https://cheerio.js.org/docs/advanced/extract#putting-it-all-together)and in the browser I get a 500 Internal Server Error and in the console I get [TypeError: (void 0) is not a function].

Does anyone know why it doesn't work?
I wanted to test the fromURL method if I do a fetch to the page and load it with the load method it works fine but that's not what I want.

This is my code

```
app.get("/", async (c) => {
  const $ = await cheerio.fromURL(
    "https://github.com/cheeriojs/cheerio/releases"
  );

  const data = $.extract({
    releases: [
      {
        // First, we select individual release sections.
        selector: "section",
        // Then, we extract the release date, name, and notes from each section.
        value: {
          // Selectors are executed within the context of the selected element.
          name: "h2",
          date: {
            selector: "relative-time",
            // The actual release date is stored in the `datetime` attribute.
            value: "datetime",
          },
          notes: {
            selector: ".markdown-body",
            // We are looking for the HTML content of the element.
            value: "innerHTML",
          },
        },
      },
    ],
  });
  return c.json(data);
});
```

**Comment by fb55** [comment]
How does your import look like?

**Reply by gerardocrr** [reply]
I import like this

import * as cheerio from "cheerio";
